Sources/Messenger/Response/Response.swift

```diff
@@ -0,0 +1,153 @@
+                            """)
+                        /// By default it returns the sent payload.
+                        default:
+                            message = .text(payload)
+                        }
+                    /// There is no provided payload.
+                    } else {
+                        message = .text("No payload provided by developer.")
+                    }
+                /// If it's a normal message.
+                } else if let message = event.message {
+                    /// Check if the message is empty.
+                    if message.text.isEmpty {
+                        self.message = .text("I'm sorry but your message is empty 😢")
+                    /// Check if the message has greetings.
+                    } else if message.text.hasGreetings() {
+                        /// Set the response message.
+                        self.message = .text("""
+                        Hi!
+                        This is an example on how to create a bot with Swift.
+                        If you want to see more, try to send me "buy", "sell" or "shop".
+                        """)
+                    /// Check if the message has "sell", "buy" or "shop" in its text.
+                    } else if message.text.lowercased().contains("sell") || message.text.lowercased().contains("buy") || message.text.lowercased().contains("shop") {
+                        /// Creates the payload with all the example elements.
+                        let payload = Payload(templateType: .generic, elements: Element.allExamples)
+                        /// Creates the attachment.
+                        let attachment = Attachment(type: .template, payload: payload)
+                        /// Finally creates the structured message.
+                        let structuredMessage = StructuredMessage(attachment: attachment)
+                        
+                        self.message = .structured(structuredMessage)
+                    /// It's a normal message, so reverse it.
+                    } else {
+                        self.message = .text(message.text.reversed(preserveFormat: true))
+                    }
+                /// If the message doent's exist.
+                } else if event.message == nil {
+                    message = .text("Webhook received unknown event.")
+                }
+                
+                /// Set the recipient with the sender ID.
+                recipient = Recipient(id: event.sender.id)
+                
+                let response = self
+                
+                /// Send the response to the Facebook Messenger APIs.
+                return try request.client().post("\(facebookGraphAPI)/\(messengerAPIVersion)/me/messages?access_token=\(messengerToken)", headers: ["Content-Type": "application/json"]) { messageRequest in
+                    try? messageRequest.content.encode(self)
+                }.map({ messageResponse -> Response in
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Chained function calls should be either on the same line, or one per line.](https://app.codacy.com/app/FabrizioBrancati/SwiftyBot/pullRequest?prid=3796373)